defaults/main.yml

```
@@ -7,3 +7,133 @@ jitsi_nat: false
 
 nginx_modify_server_names_hash_bucket_size: true
 nginx_server_names_hash_bucket_size: 64
+
+# When true, this role will manage the `config.json` file and use the
+# variables as provided below under `jitsi_config`.
+use_custom_jitsi_config_vars: false
```

I was hoping to start the variable name with `jitsi_`. All nginx related variables start with `nginx_`, the apt related ones with `apt_`, etc. But I see that `verb_something` is much more natural. How about shortening yours a bit:

```suggestion
managed_jitsi_config: false
```

---

Myself I always ensure variables are namespaced instead. So not `nginx_foo` but rather `nginx.foo`. The first makes for clumsy and often very long variable names. `certbot_acme_link_use_certbot` and nonense that like that. 

I'll change it into either `managed_jitsi_config` or `jitsi_managed_config`, just let me know what you prefer.

---

Ah, that would have been the way to go. I guess it's too late to change it now 🙈 `managed_jitsi_config` it is. Thanks for the hint! Much appreciated!